frontend/src/components/MainFeed/MainFeed.js

```diff
@@ -165,7 +165,8 @@ function MainFeed(props) {
         <React.Fragment>
           <div className="Feed">
             <div className="TopInstructions">
-              <h5 style={{ margin: '0' }}>Feed {parseInt(givenArguments.page) + 1} out of 5, please read and interact with it like your regular feed.</h5>
+              <h5 style={{ margin: '0.3em' }}>Feed {parseInt(givenArguments.page) + 1} of 5</h5> 
```

Comment:
Would it be possible to calculate the total number of feeds instead of having it hard-coded in the GUI? This way if we decide to do a different number of feeds the GUI would not need to be updated.

Reply:
I believe the backend would need to be updated to pass the total pages as a parameter if we would like this to be dynamic, otherwise, the frontend would not know the total amount.

Reply:
OK, let's discuss this in a proper issue. In the meanwhile this pull request can be merged.